test/without-slow-down.test.js

```diff
@@ -0,0 +1,36 @@
+import t from 'tap'
+import Fastify from 'fastify'
+import slowDownPlugin from '../index.js'
+import { sleep } from '../lib/helpers.js'
+import { HEADERS } from '../lib/constants.js'
+
+t.test('Should works without delay after 1 second', async t => {
+  // create the fastify server
+  const fastify = Fastify()
+  await fastify.register(slowDownPlugin)
+  fastify.get('/', async () => 'Hello from Fastify!')
```

Comment:
avoid responding this, this is not an "hello fastify" plugin, no need to reply with this